src/Services/Geocoding.js

```diff
@@ -17,11 +18,22 @@ EsriLeafletGeocoding.Services.Geocoding = Esri.Services.Service.extend({
   },
 
   suggest: function(){
-    if(this.options.url !== EsriLeafletGeocoding.WorldGeocodingService && console && console.warn){
+    if(this.options.url !== EsriLeafletGeocoding.WorldGeocodingService && !this.options.supportsSuggest && console && console.warn){
       console.warn('Only the ArcGIS Online World Geocoder supports suggestions');
```

Comment:
@jgravois since `supportsSuggest ` will be falsey the first time `suggest` is called this will always get logged. You could pass a callback through `_confirmSuggestSupport` but I would honestly rather just remove this.